15/umbraco-cms/fundamentals/data/defining-content/default-document-types.md

```diff
@@ -1,41 +1,39 @@
+An Element Type is a Document Type *without a template* designed for reusabale and repeatable set of properties. These are primarily used in editors like the Block List Editor or Block Grid Editor to create structured, nested content.
 
-Element Types cannot be used to create content in the Content tree. When you create an Element type, it automatically sets the **Is Element Type** flag to **True** on the **Permissions** tab.
+Element Types are not part of the Content tree and cannot render directly on the front end. When created, the **Is an Element Type** flag in the **Permissions** tab is automatically set to **True**.
 
 ![Element type](/15/umbraco-cms/fundamentals/data/images/element-type.png)
 
-Element Types are created using the same workflow as regular Document Types but usually contain fewer properties. You can also create Element Types as part of configuring a Block Grid or Block List Data Type.
+Use an Element Type when defining building blocks for complex page layouts, such as grid blocks or call-to-action sections. They are an essential part of modular content design.
 
 ## Folder
 
-Folders are a special type of Document Type that can be used to organize content in the Content tree. Folders can contain other content items, such as other folders or content nodes. They are useful for organizing content in a logical hierarchy, making it easier to manage and navigate your website's content. They cannot be used to create content displayed on the front end of your website.
-
-## Compositions
-
-Compositions provide a way to create reusable sets of properties that can be added to one or more Document Types. This can help simplify the management and consistency of content types across your website. Compositions can be used to define common properties shared across multiple Document Types, such as metadata fields or social media links.
-
-To get started with compositions, you will first have to create the needed Document Types as described above. Later you can take advantage of nesting and use compositions by clicking on "**Compositions**..." option on the Document Type editor. Here you will be able to select the Document Types you want to use as compositions for the current Document Type. The fields of the selected compositions will hereafter be available on the current Document Type.
-
-![Create group](/15/umbraco-cms/fundamentals/data/images/createGroup_new.png)
-{% hint style="warning" %}
-
-If you create 2 compositions that contain some common properties it is only possible to pick one of the compositions in a Document Type. If preferred, those compositions that cannot be used can be marked as hidden by check marking the `Hide unavailable options`.
+A Folder is a special type of Document Type used solely for organizing content within the Content tree. Folders can contain other content items, such as other folders or content nodes.
```

Comment:
I realize this is not something you've written, but the information about the Folder is wrong.

This type of Folder is used to organize the Document Types, and not content. The Folder cannot be used anywhere else but in the Document Types structure in the Settings section. So, you could create a Folder called "Compositions" and then create all your Composition Document Types within that folder. It will have no effect on the Content what so ever, it will only serve the purpose of organizing the Document Types more neatly.

Not sure why we never caught this before 😅 But it definitely needs to be changed!

Reply:
I've changed the description based on your comment. Can you take a look at it? Also, the Folder section is not mentioned in the v10 and 13 articles, so I've made the changes only to v14 and 15 articles.